app/models.py

```diff
@@ -109,9 +114,16 @@ def delete_event(self):
         db.session.delete(self)
         db.session.commit()
 
-    def __repr__(self):
-        """Returns a representaion of the Event class instance."""
-        return "<Event: {}>".format(self.name)
+    def serialize(self):
+        """Returns the event as a dictionary."""
+        return {
+            'name': self.name,
+            'description': self.description,
+            'category': self.category,
+            'date': self.date,
+            'author': self.author,
+            'location': self.location
+        }
 
 
 class Rsvp(db.Model):
```

Comment:
just a refactor here, how about moving the name, email and phone_no attributes to the `User` model then define a relationship to get the same?